document/presentation/main.tex

```diff
@@ -114,9 +113,11 @@
   \item An IP core can be integrated in an IOb-SoC if it provides the following X files: 
     \begin{enumerate}
     \item hardware/hardware.mk
-    \item software/embedded/embedded.mk
-      \item JTS: many more files are needed. Can you figure this out from the build (makefiles)
+    \item software/software.mk
     \end{enumerate}
+    \begin{itemize}
+      \item[--] Please refer to the files {\tt hardware/simulation/simulation.mk} and {\tt hardware/hardware.mk} in IOb-SoC for specific configurations (not necessary for this tutorial)
```

Comment:
Please refer to  files {\tt hardware/simulation/simulation.mk} and {\tt hardware/hardware.mk} in the iob-timer submodule to learn how to organize a peripheral core.

(é necessário neste tutorial)